pom.xml

```diff
@@ -0,0 +1,137 @@
+            <groupId>org.ops4j.pax.logging</groupId>
+            <artifactId>pax-logging-api</artifactId>
+            <version>${pax-logging-api.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.wso2.carbon</groupId>
+            <artifactId>org.wso2.carbon.utils</artifactId>
+            <version>${carbon.kernel.version}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven-compiler-plugin.version}</version>
+                <inherited>true</inherited>
+                <configuration>
+                    <encoding>UTF-8</encoding>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <version>${maven-bundle-plugin.version}</version>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <private-package>org.wso2.carbon.hash.provider.pbkdf2.internal</private-package>
+                        <Export-Package>
+                            !org.wso2.carbon.hash.provider.pbkdf2.internal,
+                            org.wso2.carbon.hash.provider.pbkdf2.*
+                        </Export-Package>
+                        <Import-Package>
+                            javax.crypto,javax.crypto.spec,
+                            org.apache.commons.logging; version = "${import.package.version.commons.logging}",
+                            org.osgi.framework; version = "${osgi.framework.imp.pkg.version.range}",
+                            org.osgi.service.component; version = "${osgi.service.component.imp.pkg.version.range}",
+                            org.wso2.carbon.user.core.*; version = "${carbon.kernel.package.import.version.range}"
+                        </Import-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
+
+    <properties>
+        <import.package.version.commons.logging>[1.2,2)</import.package.version.commons.logging>
+        <osgi.framework.imp.pkg.version.range>[1.7,2)</osgi.framework.imp.pkg.version.range>
+        <osgi.service.component.imp.pkg.version.range>[1.2,2)</osgi.service.component.imp.pkg.version.range>
+
+        <org.apache.felix.scr.ds-annotations.version>1.2.10</org.apache.felix.scr.ds-annotations.version>
+        <carbon.kernel.version>4.6.2-SNAPSHOT</carbon.kernel.version>
```

Comment:
I think we need to change the kernel version, once the kernel PR is merged. Lets track that.